### ISSUE TITLE
🐛  Grouped imports are split up

### ISSUE THREAD

**Issue by sschneider-ihre-pvs** [NicolasRoehm/angular-vscode-cleaner#4]
`import { computed, ref, toRefs, watchEffect } from 'vue';`
transforms to
```
import { computed }                               from 'vue';
import { ref }                                    from 'vue';
import { toRefs }                                 from 'vue';
import { watchEffect }                            from 'vue';
```

**Comment by sschneider-ihre-pvs** [comment]
this might be intended but is usually not best practice

**Comment by NicolasRoehm** [comment]
Exactly, it's done on purpose and I understand that this is generally not the best practice...
The first versions of this extension are opinionated but it can be awesome to add a configuration key, like `groupByOrigin : boolean` to split or group modules by origin.

I don't know if I can code it in a near future but I can help if you want to create a PR!

**Comment by sschneider-ihre-pvs** [comment]
I don't have any expertise in debugging or developing vscode extensions otherwise I would help.

**Comment by NicolasRoehm** [comment]
I was able to quickly handle such a use case but it's still not perfect!
You can now add `"classify-ts-import.import.groupByOrigin": true,`  into you VSCode settings.
The extension will simply ignore grouped imports but will not merge already separated imports...

Let me know if this quick fix is ok for you :)
